python/src/pipeline/pipeline.py

```diff
@@ -1452,7 +1463,7 @@ def fill_slot(self, filler_pipeline_key, slot, value):
         value_text = db.Text(encoded_value)
       else:
         # The encoded value is too big. Save it as a blob.
-        value_blob = _write_json_blob(encoded_value)
+        value_blob = _write_json_blob(encoded_value, str(filler_pipeline_key))
```

Comment:
Can we use filler_pipeline_key.name()?  It looks like that is what pipeline_id uses:
https://github.com/GoogleCloudPlatform/appengine-pipelines/blob/master/python/src/pipeline/pipeline.py#L474